packages/salesforcedx-vscode-visualforce/package.nls.json

```diff
@@ -1,4 +1,5 @@
 {
+  "configuration.title": "Salesforce Visualforce Configuration",
```

Comment:
This file follows a different convention since the original language server (the one from VS Code) uses the . convention instead of the _ convention.